app/javascript/chat/chat.jsx

```diff
@@ -1745,13 +1762,16 @@ export default class Chat extends Component {
       channelUsers,
       memberFilterQuery,
     } = this.state;
+
     const filterRegx = new RegExp(memberFilterQuery, 'gi');
+
     return (
       <div
```

Comment:
Semantically, this would make more sense as an unordered list, i.e. `<ul><li>...</li>...</ul>` and that real buttons are used instead of a `<div />` with a `role` attribute with the value `"button"`, but I will create a separate issue for that as it is not related to the bug fix. It only caught my attention because of the newly added `data-testid` attribute.